File: public/phase-2-20160620022203/JAVA/binsequence.java
// Name : BinSequence
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : length
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());


		// write your code here
		// store your results in `sequence`

		// output
		// Dummy Data
		int sequence[] = new int [] {0,1};

		System.out.println(join_string(Arrays.toString(sequence)));

	}
	public static String join_string(String inp) {
		return inp.replace("[","").replace("]","").replace(",","");
	}
}

File: public/phase-2-20160620022203/C/binsequence.c
// Name : BinSequence
#include <stdio.h>
int main() {
	// input:
	// n : length
	int n, sequence[100]={0,1}, index;
	scanf("%d", &n);


	// write your code here
	// store your results in `sequence`

	// output
	for(index = 0; index< n; index++) {
		printf("%d ",sequence[index]);
	}
	return 0;
}